scripts/start.sh

```diff
@@ -8,11 +8,6 @@ fi
 
 export $(cat .env | grep NODE_ENV | xargs)
 
-if [ "$CI" != "true" ]; then
```

Comment:
We want to run this in our Docker containers, but not on Circle _or_ on Heroku. The issue was this was getting run on the Heroku deploys, which isn't what we need.